avocado/plugins/yaml_to_mux.py

```diff
@@ -255,6 +255,8 @@ def configure(self, parser):
                              " multiplex (.yaml) FILE(s) (order dependent)")
 
     def run(self, args):
+        if getattr(args, 'remote_execution', False):
+            return
```

Comment:
This might actually not work as there is no plugins order. The `remote_execution` could be set after the yaml is processed (or before, as I said, there is no order defined).